go.mod

```diff
@@ -49,7 +49,7 @@ require (
 	github.com/docker/docker v0.7.3-0.20190327010347-be7ac8be2ae0
 	github.com/docker/go-connections v0.3.0
 	github.com/docker/go-units v0.3.3
-	github.com/docker/libnetwork v0.0.0-20180830151422-a9cd636e3789
+	github.com/docker/libnetwork v0.8.0-dev.2.0.20190624125649-f0e46a78ea34
```

Comment:
Bumped this to the version bumps netlink to v1.0.0 https://github.com/docker/libnetwork/commit/f0e46a78ea34